apps/web/src/components/score/ScoreConfig.ts

```diff
@@ -0,0 +1,57 @@
+export const mobileData = {
+  profile: {
+    padding: `15px 0`,
+    justifyContent: 'center',
+    containerJustifyContent: 'flex-start',
+    maxWidth: '20ch',
+    fontSize: '90%',
+    fontWeight: '',
+    textAlign: 'center',
+    textMarginLeft: '',
+    margin: '40px 0 0 0',
+  },
+  info: {
+    columns: 'repeat(2, 1fr)',
+    justifyContent: 'center',
+    margin: '15px 0 0 0',
```

Comment:
I think you can replace it to margin-top: 15px',
https://developer.mozilla.org/pt-BR/docs/Web/CSS/margin